data/language/dutch.txt

```diff
@@ -1171,13 +1171,13 @@ STR_1169    :(Geen)
 STR_1170    :{STRING}
 STR_1171    :{RED}Gesloten - - 
 STR_1172    :{YELLLOW}{STRINGID} - - 
-STR_1173    :{SMALLFONT}{BLACK}Voetpaden en wachtrijen neerzetten
+STR_1173    :{SMALLFONT}{BLACK}Voetpaden en wachtrijen aanleggen
 STR_1174    :Banier in de weg
 STR_1175    :Kan dit niet op hellend voetpad neerzetten
-STR_1176    :Kan hier geen voetpad neerzetten...
+STR_1176    :Kan hier geen voetpad aanleggen...
 STR_1177    :Kan dit voetpad niet verwijderen...
 STR_1178    :Landhelling is ongeschikt
-STR_1179    :Voetpad staat in de weg
+STR_1179    :Voetpad ligt in de weg
```

Comment:
Evenzo is het dan weer waar dat een voetpad op de grond niet staat, beide vormen zijn niet overal gepast. Misschien is 'Voetpad is in de weg' beter?